tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build"
```

Comment:
nit: can we call it `dist` instead of `build`? Just as that's what it's named in all my other repos and it'll trip me up when context switching 😄

Reply:
sure! done.